nibabel/cifti2/cifti2.py

```diff
@@ -0,0 +1,946 @@
+        vertices.attrib['BrainStructure'] = str(self.brain_structure)
+
+        if self.vertices is not None:
+            vertices.text = ' '.join(self.vertices.astype(str))
+        return vertices
+
+
+class Cifti2Parcel(object):
+    """Cifti2 parcel: association of a name with vertices and/or voxels
+
+    Attributes
+    ----------
+    name : str
+        Name of parcel
+    voxel_indices_ijk : None or Cifti2VoxelIndicesIJK
+        Voxel indices associated with parcel
+    vertices : list of Cifti2Vertices
+        Vertices associated with parcel
+    """
+    def __init__(self, name=None, voxel_indices_ijk=None, vertices=None):
+        self.name = name
+        self.voxel_indices_ijk = voxel_indices_ijk
+        self.vertices = vertices if vertices is not None else []
+
+    @property
+    def voxel_indices_ijk(self):
+        return self._voxel_indices_ijk
+
+    @voxel_indices_ijk.setter
+    def voxel_indices_ijk(self, value):
+        self._voxel_indices_ijk = _value_if_klass(value, Cifti2VoxelIndicesIJK)
+
+    def add_cifti_vertices(self, vertices):
+        """ Adds a vertices to the Cifti2Parcel
+
+        Parameters
+        ----------
+        vertices : Cifti2Vertices
+        """
+        if not isinstance(vertices, Cifti2Vertices):
+            raise TypeError("Not a valid Cifti2Vertices instance")
+        self.vertices.append(vertices)
+
+    def remove_cifti2_vertices(self, ith):
+        """ Removes the ith vertices element from the Cifti2Parcel """
+        self.vertices.pop(ith)
+
+    def _to_xml_element(self):
+        assert self.name is not None
+        parcel = xml.Element('Parcel')
+        parcel.attrib['Name'] = str(self.name)
+        if self.voxel_indices_ijk:
+            parcel.append(self.voxel_indices_ijk._to_xml_element())
+        for vertex in self.vertices:
+            parcel.append(vertex._to_xml_element())
+        return parcel
+
+
+class Cifti2TransformationMatrixVoxelIndicesIJKtoXYZ(object):
+    """Matrix that translates voxel indices to spatial coordinates
+
+    Attributes
+    ----------
+    meter_exponent : int
+        "[S]pecifies that the coordinate result from the transformation matrix
+        should be multiplied by 10 to this power to get the spatial coordinates
+        in meters (e.g., if this is '-3', then the transformation matrix is in
+        millimeters)."
+    matrix : array-like shape (4, 4)
+        Affine transformation matrix from voxel indices to RAS space
+    """
+    # meterExponent = int
+    # matrix = np.array
+
+    def __init__(self, meter_exponent=None, matrix=None):
+        self.meter_exponent = meter_exponent
+        self.matrix = matrix
+
+    def _to_xml_element(self):
+        assert self.matrix is not None
+        trans = xml.Element('TransformationMatrixVoxelIndicesIJKtoXYZ')
+        trans.attrib['MeterExponent'] = str(self.meter_exponent)
+        trans.text = '\n'.join(' '.join(map('%.10f'.format, row))
```

Comment:
@satra  I think here you mean 
`map('{:.10f}'.format, row)`
as the previous syntax does not work